gnosis/helper.js

```diff
@@ -0,0 +1,13 @@
+
+async function submitTransactionToGnosisSafe(DEPLOYER_SAFE, artifact, functionName, ...args) {
```

Comment:
This is excellent.

This is exactly what we needed `const tx = await artifact.connect(signer)[functionName](...args)` 👍